src/features/game/types/withdrawables.test.ts

```diff
@@ -310,6 +310,27 @@ describe("withdrawables", () => {
       const enabled = BUMPKIN_WITHDRAWABLES["Infernal Pitchfork"](state);
       expect(enabled).toBeFalsy();
     });
+    it("prevents withdrawal of Crimstone Hammer 7 days after last crimstone mine", () => {
```

Comment:
Is this description correct? Should it read something like:
```suggestion
    it("prevents withdrawal of Crimstone Hammer if less than 7 days after last crimstone mine", () => {
```